includes/class-sensei-quiz.php

```diff
@@ -2519,17 +2519,36 @@ private static function maybe_get_button_html_for_quiz_footer( $lesson_id = null
 	/**
 	 * Returns the HTML for a primary button anchor.
 	 *
-	 * @param string $button_text The button text.
-	 * @param string $url         The URL.
+	 * @param string      $button_text The button text.
+	 * @param string|null $url         The URL.
+	 * @param array       $classes     CSS classes to add to the button.
 	 *
 	 * @return string The HTML for the primary button anchor.
 	 */
-	private static function get_primary_button_anchor_html( $button_text, $url ) {
-		return '<a class="wp-element-button sensei-course-theme__button is-primary" href="' . esc_url( $url ) . '">' .
-						esc_html( $button_text ) .
-				'</a>';
-	}
+	public static function get_primary_button_html( $button_text, $url = null, $classes = [] ) {
+		$href    = '';
+		$classes = array_merge(
+			array(
+				'wp-block-button__link',
+				'wp-element-button',
+				'sensei-course-theme__button',
+				'is-primary',
+			),
+			$classes
+		);
 
+		if ( $url ) {
+			$href = ' href="' . esc_url( $url ) . '"';
+		}
+
+		return (
```

Comment:
@renatho That is weird. I fixed it in https://github.com/Automattic/sensei/pull/7210/commits/a6724cbf7677980c5d41efb60da48512433ac345, but I don't quite understand why it worked when that other button was added. In that case, these styles are already coming from the page itself, but are not there when the extra button is not there. I even tried adding the same markup / CSS classes in case that triggered the inline CSS, but no luck:

![Screenshot 2023-10-30 at 12 34 07 PM](https://github.com/Automattic/sensei/assets/1190420/e02062f2-79a5-4ea8-82f6-46fe81dff146)